pkg/operatormetrics/collector.go

```diff
@@ -68,11 +68,17 @@ func collectValue(ch chan<- prometheus.Metric, metric Metric, cr CollectorResult
 	}
 
 	labels := map[string]string{}
+
 	for k, v := range cr.ConstLabels {
-		labels[k] = v
+		if v != "" {
```

Comment:
what about "<none>" values? in many of our metric we use none to set empty labels labels

Reply:
but "<none>" doesnt make sense for a library
we should change in the kubevirt code after from none to empty